src/common/marlin_server.cpp

```diff
@@ -71,12 +71,12 @@ typedef struct _marlin_server_t {
     uint8_t pqueue_tail;                                     // copy of planner.block_buffer_tail
     uint8_t pqueue;                                          // calculated number of records in planner queue
     uint8_t gqueue;                                          // copy of queue.length - number of commands in gcode queue
+    uint8_t server_device_state;                             // device state stored by server
     uint32_t command;                                        // actually running command
     uint32_t command_begin;                                  // variable for notification
     uint32_t command_end;                                    // variable for notification
     marlin_mesh_t mesh;                                      // meshbed leveling
     uint64_t mesh_point_notsent[MARLIN_MAX_CLIENTS];         // mesh point mask (points that are not sent)
-    uint8_t device_state_change;                             // marks that change in device's printing state has occured
```

Comment:
no need of additional mask, there is an array of event masks.
it should be same as how file change mask used

Reply:
commit 0cb14560a...